uia.gemspec

```diff
@@ -25,13 +25,13 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ['lib']
 
-  spec.add_runtime_dependency 'ffi', '~> 1.9.4'
-  spec.add_runtime_dependency 'require_all'
+  spec.add_runtime_dependency 'ffi', '~> 1.11.2'
```

Comment:
Only thing I'd be cautious about is that `ffi` can vary from version to version particularly with calling system calls that are `cdecl` vs. `stdcall`? So if this suite passes + using this version of the gem with some other test suite seems good then it's probably 👍 ?

Just curious if you bumped it for any particular reason or if it was just latest or something.